Loader.server.lua

```diff
@@ -22,6 +22,9 @@ local Config = {
     --Default rank a new player would have.
     DefaultAdminLevel = -1,
 
+    --Default Admin Level for Server Lock (slock)
+    DefaultServerLockAdminLevel = 0,
```

Comment:
While I like this being configurable, I don't like it being at the top-level. More may come in the future and make the configuration very long. This is something I would be fine with (feel free to suggest an alternative name).

```lua
CommandConfigurations = { --CommandSettings, CommandOverrides, or something else would be fine.
    DefaultServerLockAdminLevel = 0,
},
```

With that being said, I am aware that it makes the logic in `Configuration.lua` a bit tricky, since you have to correct this being `nil` *and* `DefaultServerLockAdminLevel` being `nil`.

Reply:
I had initially put it there since the `DefaultAdminLevel` and `Admins` Configuration is right next to it, and didn't know where else to put it.

For `Configuration.lua` a possible way to address this (based on what I initially thought of) would be having a constant at the top of the script that is the defaults for `CommandConfigurations` and run a loop on the reference settings and compare to the configuration settings, and make changes accordingly.

```lua
local COMMAND_CONFIGURATIONS = {
    --Default Admin Level for Server Lock (slock).
    DefaultServerLockAdminLevel = 0,

    --Other future settings accordingly.
}
```

Commenting this before actually writing any code to do this, as you (having a lot more experience than I do) may know of a more effective and efficient way.

Reply:
I didn't have the idea of putting the defaults at the top, but I like it. Only thing I'd change is to name it `DEFAULT_COMMAND_CONFIGURATIONS`.